drivers/display/display_gc9x01x.c

```diff
@@ -0,0 +1,745 @@
+static int gc9x01x_pm_action(const struct device *dev, enum pm_device_action action)
+{
+	int ret = 0;
+
+	switch (action) {
+	case PM_DEVICE_ACTION_RESUME:
+		ret = gc9x01x_exit_sleep(dev);
+		break;
+	case PM_DEVICE_ACTION_SUSPEND:
+		ret = gc9x01x_enter_sleep(dev);
+		break;
+	default:
+		ret = -ENOTSUP;
+		break;
+	}
+
+	return ret;
+}
+#endif /* CONFIG_PM_DEVICE */
+
+/* Device driver API*/
+static const struct display_driver_api gc9x01x_api = {
+	.blanking_on = gc9x01x_display_blanking_on,
+	.blanking_off = gc9x01x_display_blanking_off,
+	.write = gc9x01x_write,
+	.read = gc9x01x_read,
+	.get_framebuffer = gc9x01x_get_framebuffer,
+	.set_brightness = gc9x01x_set_brightness,
+	.set_contrast = gc9x01x_set_contrast,
+	.get_capabilities = gc9x01x_get_capabilities,
+	.set_pixel_format = gc9x01x_set_pixel_format,
+	.set_orientation = gc9x01x_set_orientation,
+};
+
+#define INST_DT_GC9X01X(n) DT_INST(n, galaxycore_gc9x01x)
+
+#define GC9X01X_INIT(n, t)                                                                         \
+	GC9X01X_REGS_INIT(n);                                                                      \
+	static const struct gc9x01x_config gc9x01x_config_##n = {                                  \
+		.spi = SPI_DT_SPEC_GET(INST_DT_GC9X01X(n), SPI_OP_MODE_MASTER | SPI_WORD_SET(8),   \
+				       0),                                                         \
+		.cmd_data = GPIO_DT_SPEC_GET(INST_DT_GC9X01X(n), cmd_data_gpios),                  \
+		.reset = GPIO_DT_SPEC_GET_OR(INST_DT_GC9X01X(n), reset_gpios, {0}),                \
+		.backlight = PWM_DT_SPEC_GET_OR(DT_PROP(INST_DT_GC9X01X(n), backlight_pwm), 0),    \
+		.pixel_format = DT_PROP(INST_DT_GC9X01X(n), pixel_format),                         \
+		.rotation = DT_PROP(INST_DT_GC9X01X(n), rotation),                                 \
+		.x_resolution = DT_PROP(INST_DT_GC9X01X(n), width),                                \
+		.y_resolution = DT_PROP(INST_DT_GC9X01X(n), height),                               \
+		.inversion = DT_PROP(INST_DT_GC9X01X(n), display_inversion),                       \
+		.regs = &gc9x01x_regs_##n,                                                         \
+		.regs_init_fn = gc9x01x_regs_init,                                                 \
+	};                                                                                         \
+	static struct gc9x01x_data gc9x01x_data_##n;                                               \
+	PM_DEVICE_DT_INST_DEFINE(inst, gc9x01x_pm_action);                                         \
+	DEVICE_DT_DEFINE(INST_DT_GC9X01X(n), gc9x01x_init, PM_DEVICE_DT_INST_GET(inst),            \
+			 &gc9x01x_data_##n, &gc9x01x_config_##n, POST_KERNEL,                      \
+			 CONFIG_DISPLAY_INIT_PRIORITY, &gc9x01x_api)
+
+#define DT_INST_FOREACH_GC9X01X_STATUS_OKAY                                                        \
+	LISTIFY(DT_NUM_INST_STATUS_OKAY(galaxycore_gc9x01x), GC9X01X_INIT, (;))
+
+#ifdef CONFIG_GC9X01X
```

Comment:
this cannot not happen, the driver depends on `CONFIG_GC9X01X` to be built

Reply:
fixed in [3af98ab](https://github.com/zephyrproject-rtos/zephyr/commit/3af98ab51e4a42df60a3209ef92480cd397189a3)